junifer/markers/ets_rss.py

```diff
@@ -0,0 +1,124 @@
+"""Provide class for root sum of squares of edgewise timeseries."""
+
+# Authors: Leonard Sasse <l.sasse@fz-juelich.de>
+#          Nicolás Nieto <n.nieto@fz-juelich.de>
+#          Sami Hamdan <s.hamdan@fz-juelich.de>
+#          Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+from typing import Dict, List
+
+import numpy as np
+from scipy.stats import zscore
+
+from ..api.decorators import register_marker
+from ..utils import logger
+from .base import BaseMarker
+from .parcel import ParcelAggregation
+
+
+@register_marker
+class RSSETSMarker(BaseMarker):
+    """Class for root sum of squares of edgewise timeseries.
+
+    Parameters
+    ----------
+    atlas : str
+        The name of the atlas.
+    aggregation_method : str, optional
+        The aggregation method (default "mean").
+
+    """
+
+    def __init__(
+            self, atlas: str,
+            aggregation_method: str = "mean",
+            name: str = None,
+    ) -> None:
+        """Initialize the class."""
+        self.atlas = atlas
+        self.aggregation_method = aggregation_method
+        super().__init__(on=["BOLD"], name=name)
+
+    def get_output_kind(self, input: List[str]) -> List[str]:
+        """Get output kind.
+
+        Parameters
+        ----------
+        input : list of str
+            The input to the marker. The list must contain the
+            available Junifer Data dictionary keys.
+
+        Returns
+        -------
+        list of str
+            The updated list of output kinds, as storage possibilities.
+
+        """
+        outputs = []
+        for t_input in input:
+            if t_input in ["BOLD"]:
+                outputs.append("timeseries")
+            else:
+                raise ValueError(f"Unknown input kind for {t_input}")
+        return outputs
+
+    # TODO: complete type annotations
+    def store(self, kind: str, out, storage) -> None:
+        """Store.
+
+        Parameters
+        ----------
+        kind
+        out
+        storage
+
+        """
+        logger.debug(f"Storing {kind} in {storage}")
+        if kind in ["BOLD"]:
+            storage.store_timeseries(**out)
```

Comment:
Include this in coverage, otherwise we don't know if it works.

Reply:
The `store_timeseries()` method fails as it's not implemented, or did you implement it?

Reply:
I do want to make a small change so that the edge time series is computed in a little helper function _ets() that takes an ndarray and returns an ndarray, because I also need this for the EdgeCentricFC marker, which I will then put in the same file, which I want to rename ets.py perhaps.